docs/docs/guides/visualizing-your-results.mdx

```diff
@@ -44,17 +44,17 @@ The Table option is good for looking at (surprise, surprise) tabular data, or fo
 
 ![table chart](./assets/table-chart.png)
 
-### Column chart
+### Bar chart
 
-Column charts are helpful to compare things between different groups (e.g. the number of orders you have by product type) or to track how a number changes over time if you have a smaller number of x-axis values (e.g. number of new users per month over a year). 
+Bar charts are helpful to compare things between different groups (e.g. the number of orders you have by product type) or to track how a number changes over time if you have a smaller number of x-axis values (e.g. number of new users per month over a year). 
 
-![column chart](./assets/column-chart.png)
+![bar chart](./assets/column-chart.png)
```

Comment:
Missing `column` rename on like 53 ? `column graphs` should be `bar graphs` or `bar charts`.

Reply:
Great catch thanks!